src/proposals/20250821/SparkEthereum_20250821.t.sol

```diff
@@ -0,0 +1,350 @@
+            variableRateSlope1Spread : 0.0125e27,
+            variableRateSlope2       : 0.15e27,
+            optimalUsageRatio        : 0.95e27
+        });
+        _testRateTargetKinkIRMUpdate("USDT", oldParams, newParams);
+    }
+
+    function test_ETHEREUM_sparkLend_wethIrmUpdate() public onChain(ChainIdUtils.Ethereum()) {
+        RateTargetKinkIRMParams memory oldParams = RateTargetKinkIRMParams({
+            irm                      : OLD_WETH_IRM,
+            baseRate                 : 0,
+            variableRateSlope1Spread : -0.005e27,
+            variableRateSlope2       : 1.2e27,
+            optimalUsageRatio        : 0.90e27
+        });
+        RateTargetKinkIRMParams memory newParams = RateTargetKinkIRMParams({
+            irm                      : NEW_WETH_IRM,
+            baseRate                 : 0,
+            variableRateSlope1Spread : -0.003e27,
+            variableRateSlope2       : 1.2e27,
+            optimalUsageRatio        : 0.90e27
+        });
+        _testRateTargetKinkIRMUpdate("WETH", oldParams, newParams);
+    }
+
+    function test_ETHEREUM_sparkLend_marketUpdates() public onChain(ChainIdUtils.Ethereum()) {
+        SparkLendContext memory ctx = _getSparkLendContext();
+
+        ReserveConfig[] memory allConfigsBefore = createConfigurationSnapshot('', ctx.pool);
+
+        ReserveConfig memory cbBTCConfigBefore  = _findReserveConfigBySymbol(allConfigsBefore, 'cbBTC');
+        ReserveConfig memory ezETHConfigBefore  = _findReserveConfigBySymbol(allConfigsBefore, 'ezETH');
+        ReserveConfig memory lBTCConfigBefore   = _findReserveConfigBySymbol(allConfigsBefore, 'LBTC');
+        ReserveConfig memory rsETHConfigBefore  = _findReserveConfigBySymbol(allConfigsBefore, 'rsETH');
+        ReserveConfig memory tBTCConfigBefore   = _findReserveConfigBySymbol(allConfigsBefore, 'tBTC');
+        ReserveConfig memory weETHConfigBefore  = _findReserveConfigBySymbol(allConfigsBefore, 'weETH');
+        ReserveConfig memory wethConfigBefore   = _findReserveConfigBySymbol(allConfigsBefore, 'WETH');
+        ReserveConfig memory wstethConfigBefore = _findReserveConfigBySymbol(allConfigsBefore, 'wstETH');
+
+        executeAllPayloadsAndBridges();
+
+        ReserveConfig[] memory allConfigsAfter = createConfigurationSnapshot('', ctx.pool);
+
+        // wstETH Before
+
+        assertEq(wstethConfigBefore.ltv,                  79_00);
+        assertEq(wstethConfigBefore.liquidationThreshold, 80_00);
+
+        // wstETH After
+
+        ReserveConfig memory wstethConfigAfter = wstethConfigBefore;
+
+        wstethConfigAfter.ltv                  = 83_00;
+        wstethConfigAfter.liquidationThreshold = 84_00;
+
+        _validateReserveConfig(wstethConfigAfter, allConfigsAfter);
+
+        _assertBorrowCapConfig({
+            asset:            Ethereum.WSTETH,
+            max:              1,
+            gap:              1,
+            increaseCooldown: 12 hours
+        });
```

Comment:
Use this formatting and order to match the same order as the forum post for easier review

```solidity
        // wstETH

        assertEq(wstethConfigBefore.ltv,                  79_00);
        assertEq(wstethConfigBefore.liquidationThreshold, 80_00);

        ReserveConfig memory wstethConfigAfter = wstethConfigBefore;

        wstethConfigAfter.ltv                  = 83_00;
        wstethConfigAfter.liquidationThreshold = 84_00;

        _validateReserveConfig(wstethConfigAfter, allConfigsAfter);

        _assertBorrowCapConfig({
            asset:            Ethereum.WSTETH,
            max:              1,
            gap:              1,
            increaseCooldown: 12 hours
        });
```

Reply:
done